source/fe/mapping_q_generic.cc

```diff
@@ -2191,28 +2210,75 @@ MappingQGeneric<dim, spacedim>::transform_points_real_to_unit_cell(
   const DerivativeForm<1, spacedim, dim> A_inv =
     affine_factors.first.covariant_form().transpose();
 
-  for (unsigned int i = 0; i < real_points.size(); ++i)
-    {
-      try
```

Comment:
What happened with the try-catch?

Reply:
We should not need the `try-catch` any more because the implementation in the class never throws (at least not the `MappingFailed` exception).